docs/development/sql_api.rst

```diff
@@ -0,0 +1,78 @@
+:orphan:
```

Comment:
`docs/development/using_sql_on_ray.rst` has been a leftover, which should have been removed, thanks for finding and removing it! I think the content of this file should combined with `docs/usage_guide/advanced_usage/modin_sql.html` and this file should be removed. Also, [this section](https://modin.readthedocs.io/en/stable/usage_guide/advanced_usage/index.html#sql-on-modin-dataframes) should be extended a bit and the picture should be replaced with a code snippet, for example, as it is not up to date (we don't have `modin.sql` module for quite a long time).

Reply:
We can do that as part of a separate issue to not bloat up this PR.